lms/static/js/spec/verify_student/make_payment_step_view_ab_testing_spec.js

```diff
@@ -0,0 +1,221 @@
+            });
+
+            it( 'A/B Testing: provides working payment buttons for a single processor', function() {
+                createView({processors: ['cybersource']});
+                checkPaymentButtons( AjaxHelpers.requests(this), {cybersource: "Checkout"});
+            });
+
+            it( 'A/B Testing: provides working payment buttons for multiple processors', function() {
+                createView({processors: ['cybersource', 'paypal', 'other']});
+                checkPaymentButtons( AjaxHelpers.requests(this), {
+                    cybersource: "Checkout",
+                    paypal: "Checkout with PayPal",
+                    other: "Checkout with other"
+                });
+            });
+
+            it( 'A/B Testing: by default minimum price is selected if no suggested prices are given', function() {
+                var view = createView(),
+                    requests = AjaxHelpers.requests( this );
+
+                expectPriceSelected( STEP_DATA.minPrice);
+                expectPaymentButtonEnabled( true );
+
+                goToPayment( requests, {
+                    amount: STEP_DATA.minPrice,
+                    courseId: STEP_DATA.courseKey,
+                    processor: STEP_DATA.processors[0],
+                    succeeds: true
+                });
+                expectPaymentSubmitted( view, {foo: 'bar'} );
+            });
+
+            it( 'A/B Testing: min price is always selected even if contribution amount is provided', function() {
+                // Pre-select a price NOT in the suggestions
+                createView({
+                    contributionAmount: '99.99'
+                });
+
+                // Expect that the price is filled in
+                expectPriceSelected( STEP_DATA.minPrice );
+            });
+
+            it( 'A/B Testing: disables payment for inactive users', function() {
+                createView({ isActive: false });
+                expectPaymentDisabledBecauseInactive();
+            });
+
+            it( 'A/B Testing: displays an error if the order could not be created', function() {
+                var requests = AjaxHelpers.requests( this ),
+                    view = createView({});
+
+                goToPayment( requests, {
+                    amount: STEP_DATA.minPrice,
+                    courseId: STEP_DATA.courseKey,
+                    processor: STEP_DATA.processors[0],
+                    succeeds: false
+                });
+
+                // Expect that an error is displayed
+                expect( view.errorModel.get('shown') ).toBe( true );
+                expect( view.errorModel.get('errorTitle') ).toEqual( 'Could not submit order' );
+                expect( view.errorModel.get('errorMsg') ).toEqual( SERVER_ERROR_MSG );
+
+                // Expect that the payment button is re-enabled
+                expectPaymentButtonEnabled( true );
+            });
+
+        });
+    }
+);
```

Comment:
Please update this document to use quotation marks consistently (i.e. replace " with ' wherever possible).